README.md

```diff
@@ -610,14 +611,14 @@ A startup goes through [several stages of growth](#stages-of-a-startup) as it ra
 based on the hope and expectation that the company will grow and make more money in the
 future.
 
-- 🄳 Companies add (or “issue”) shares during fundraising, which can then be awarded to
+- 🄳 Companies add (or issue) shares during fundraising, which can then be awarded to
   investors in exchange for their investment.
   As the number of outstanding shares goes up, the percentage ownership of each shareholder
   goes down. This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp).
-  - ☝️ Dilution doesn’t necessarily mean that you’re losing anything as a shareholder.
-    As a company issues stock and raises money, the smaller percentage of the company you *do*
-    have could be worth more.
-    The size of your slice gets relatively smaller, but, if the company is growing, the size
+  - ☝️ Dilution doesn’t necessarily mean that shareholders lose anything.
```

Comment:
Good either way to me. Just want to clarify, don't think there's any problem with "you" when it's an informal note or explanatory prose (as opposed to definitions).

Reply:
Sure. Reverted to you.